go/private/common.bzl

```diff
@@ -208,7 +208,7 @@ def has_versioned_shared_lib_extension(path):
 
     return True
 
-MINIMUM_BAZEL_VERSION = "2.2.0"
+MINIMUM_BAZEL_VERSION = "3.0.0"
```

Comment:
Please also update the minimum version in README.rst.

Reply:
Okay, I wasn't sure if I should do that, since it specifically references the latest released version of rules_go.

Reply:
You're right, it's actually better to leave that for the next minor release. Forget my earlier comment.